### ISSUE TITLE
Docs: Check type for long form of integer and boolean

### ISSUE THREAD

**Issue by GaryJones** [WordPress-Coding-Standards/WordPress-Coding-Standards#666]
The short form of `int` and `bool` is preferred for `@param` and `@return`, but currently:

```php
/**
 * ...
 *
 * @param integer $object_id Object ID.
 * @param bool    $single    Whether returns as single or array.
 */
```
is not reported.

There may be something in PHPCS that covers this already that just needs adding in.

**Comment by GaryJones** [comment]
Relevant: https://github.com/squizlabs/PHP_CodeSniffer/pull/1047
